docs/_docs/reference/constraint.md

```diff
@@ -188,3 +188,104 @@ trait RefinedTypeOps[A, C, T]:
   def option(value: A): Option[T] =
     Option.when(rtc.test(value))(value.asInstanceOf[T])
 ```
+
+# Pre-defined constraints
+
+Iron provides a set of pre-defined constraints in the `io.github.iltotore.iron.constraint` package.
+You can find them in the [API documentation](https://iltotore.github.io/iron/api/io/github/iltotore/iron/constraint/index.html).
+
+## Global constraints
+
+Some constraints are available for all types.
+They are located in the `io.github.iltotore.iron.constraint.any` object.
```

Comment:
What about using `[[package.name]]` instead of backquotes to link it to the Scaladoc?

Reply:
You're right, i'm fixing this